src/CommandPool.php

```diff
@@ -49,7 +49,7 @@ public function __construct(
         if (!isset($config['concurrency'])) {
             $config['concurrency'] = 25;
         }
-        if (!array_key_exists('preserve_iterator_keys', $config)) {
+        if (empty($config['preserve_iterator_keys'])) {
```

Comment:
Apologies for this quick redux, I meant in the other comment to remove this whole conditional setting of `'preserve_iterator_keys'` and replace the conditional in the `$mapFn` loop below with the `empty()` check. Sorry for the confusion!

Reply:
Aha. Yep!